src/python/pants/bin/daemon_pants_runner.py

```diff
@@ -185,8 +201,9 @@ def post_fork_child(self):
     # Set context in the process title.
     set_process_title('pantsd-runner [{}]'.format(' '.join(self._args)))
 
-    # Broadcast our pid to the remote client so they can send us signals (i.e. SIGINT).
-    NailgunProtocol.send_pid(self._socket, bytes(os.getpid()))
+    # Broadcast our process group ID (in PID form - i.e. negated) to the remote client so
```

Comment:
If this is negated into a process group, is it referred to as something else? A `PGID` or something?

Reply:
no, the vanilla form would be a PGID. the negated form is PID form.

os.kill(PGID) -> error
os.kill(PGID_IN_PID_FORM) -> works
os.killpg(PGID) -> works

we could invert to PGID form throughout the stack, but this seemed slightly less intrusive in general.